includes/controls/media.php

```diff
@@ -210,6 +210,7 @@ protected function get_default_settings() {
 			'label_block' => true,
 			'media_type' => 'image',
 			'should_include_svg_inline_option' => false,
+			'library_type' => '',
```

Comment:
Why?

Reply:
Because I've added a `library_type` support in the JS, so I set a default value here.